test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js

```diff
@@ -975,6 +975,28 @@ describe( `[${ host }] Calypso Gutenberg Site Editor Tracking: (${ screenSize })
 
 				assert( isBackToDashboardLocated );
 			} );
+
+			it( 'should track "wpcom_block_editor_nav_sidebar_item_edit" when switching to a content item', async function () {
+				const pagesMenuItemLocator = By.css(
+					'[role="region"][aria-label="Navigation Sidebar"] [role="menu"] [title="Pages"] button'
+				);
+				await driverHelper.clickWhenClickable( this.driver, pagesMenuItemLocator );
+
+				const contentMenuItemLocator = By.css(
+					'[role="region"][aria-label="Navigation Sidebar"] [role="menu"] [title="Home"] button'
+				);
+				await driverHelper.clickWhenClickable( this.driver, contentMenuItemLocator );
+
+				const eventsStack = await getEventsStack( this.driver );
+				const editEvents = eventsStack.filter(
+					( [ eventName ] ) => eventName === 'wpcom_block_editor_nav_sidebar_item_edit'
+				);
+				// TODO: Change this to 1 once https://github.com/WordPress/gutenberg/pull/33286 fix is deployed
+				assert.strictEqual( editEvents.length, 2 );
```

Comment:
Another idea might be to debounce the function that sends the event with a very small time window so only 1 event sends.  Although, this double-dispatch should definitely be fixed anyways and will solve this problem.

Reply:
[Debouncing!](https://github.com/Automattic/wp-calypso/pull/54416)